.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 21
```

Comment:
This will be enforced to be a lot lower than that...
That's far too long for a PR to be open